spec/controler_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'controler'
+require 'user_interface'
+
+RSpec.describe Controler do
+  let(:actions) { [double('NullAction')] }
+  let(:output) { StringIO.new }
+  let(:valid_input) { StringIO.new('1') }
+
+  it 'tells the user_interface to print the menu' do
```

Comment:
`user_interface` (with an underscore) is an implementation detail
test descriptions are documentation, so should make no reference to variable names and be written in plain English :+1: